### PR TITLE
Fix/pin panic

### PR DESCRIPTION
**Pull request by snissn**
Swap out the GOB encoder for msg pack as this doesn't panic. 
Also change the working directory for the queue since the data format is different now, so we don't have collisions

### CODE REVIEW

File: pinner/pinmgr.go [hunk 1]
```diff
@@ -356,8 +366,11 @@ func (pm *PinManager) enqueuePinOp(po *PinningOperation) {
 	if po.SkipLimiter {
 		u = 0
 	}
-
-	_, err = pm.pinQueue.EnqueueObject(getUserForQueue(u), po)
```
Comment (en0ma) [inline pinner/pinmgr.go:360]: @snissn can we add some form of validation on the addrs before enqueuing, so we don't fail and lose the message? Let's validate, if it fails validation, set the address to `[]`, so we still process the message, and ignore the bad addrs.

Comment (alvin-reyes) [reply pinner/pinmgr.go:360]: Yes. Lets prioritize this today and get it merged asap. I’d like to do an emergency release with this fix.

Comment (alvin-reyes) [reply pinner/pinmgr.go:360]: I’m OK with that. @en0ma?

Let's get this merged.

Comment (en0ma) [reply pinner/pinmgr.go:360]: ok

**Review by en0ma — APPROVED** [review]

**Pull request merged**